src/main/java/success/planfit/domain/user/User.java

```diff
@@ -0,0 +1,52 @@
+package success.planfit.domain.user;
+
+
+import jakarta.persistence.*;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+
+@Getter
+@NoArgsConstructor
+@Entity
+@DiscriminatorColumn(name="authorized_by", discriminatorType = DiscriminatorType.STRING)
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column(nullable = false)
+    private String name;
+
+    private String phone_number;
```

Comment:
```suggestion
    private String phoneNumber;
```

MySQL에서 사용하는 관례인 스네이크 케이스로 필드명을 작성해 주신 것 같아요!
하지만 SpringBoot는 기본 설정으로 **카멜 케이스와 스네이크 케이스 간 변환** 기능을 지원해 줍니다.
그러므로 엔티티의 필드명들을 위 예시처럼 카멜 케이스로 수정해 주실 수 있나요?

아니면 혹시 MySQL의 컬럼명과 완전히 통일하기 위해 스네이크 케이스를 사용하는 것이 더 좋다고 생각하신 거라면, 의논할 수 있도록 댓글로 남겨주시거나 연락 주세요. :)